e2e-tests/native/test/specs/navigateToDashboard.e2e.ts

```diff
@@ -0,0 +1,31 @@
+import LandingPage from '../pageobjects/landing.page'
+import DashboardPage from '../pageobjects/dashboard.page'
+import {hideKeyboard, swipe} from "../Action";
+
+describe('navigate to dashboard', () => {
+
+  it('filter and navigate to City', async () => {
+    const filter = 'wirschaffendas'
+
+    expect(await LandingPage.exists()).toBeTruthy()
+
+    const cities = await LandingPage.cities
+    const search = await LandingPage.search
+
+    expect(cities.length).toBeGreaterThan(0)
+    await search.click()
+    await search.addValue(filter)
+
+    await hideKeyboard()
+    await swipe('DOWN')
```

Comment:
This looks kind of weird if you don't know why this is the case, I think it would be clearer to just to a swipe in hideKeyboard for iOS. Or is it needed on android as well?